fw-child/resources/js/map.js

```diff
@@ -2389,6 +2389,16 @@
           if (selected_scheme_item.data('scheme-type') === 'divergent') {
             high = Math.max(Math.abs(low), Math.abs(high));
             low = high * -1;
+            // workaround to avoid legend with repeated values for very low range variables
```

Comment:
There is also a `generate_ramp()` function in download.js. Should this code also be updated there?

Reply:
No, the download page doesn't have the custom color scheme option.